src/unit.rs

```diff
@@ -4146,10 +4172,22 @@ mod tests {
         assert_next_sibling(&mut cursor, "004");
         assert_next_dfs(&mut cursor, "005", 1);
         assert_next_sibling(&mut cursor, "006");
-        assert!(cursor.next_sibling().expect("Should parse next sibling").is_none());
-        assert!(cursor.next_sibling().expect("Should parse next sibling").is_none());
-        assert!(cursor.next_sibling().expect("Should parse next sibling").is_none());
-        assert!(cursor.next_sibling().expect("Should parse next sibling").is_none());
+        assert!(cursor
+                    .next_sibling()
+                    .expect("Should parse next sibling")
+                    .is_none());
+        assert!(cursor
+                    .next_sibling()
+                    .expect("Should parse next sibling")
+                    .is_none());
+        assert!(cursor
+                    .next_sibling()
+                    .expect("Should parse next sibling")
+                    .is_none());
+        assert!(cursor
+                    .next_sibling()
+                    .expect("Should parse next sibling")
+                    .is_none());
```

Comment:
Setting `chain_one_line_max = 100` would avoid this. I also like `fn_call_width = 100`.